pyatlan/client/constants.py

```diff
@@ -15,6 +15,7 @@
 
 ROLE_API = "roles"
 GROUP_API = "groups"
```

Comment:
The v2/groups endpoint doesn't replace all previous occurrences of groups? It's only used when retrieving groups?